src/NServiceBus.RabbitMQ/RabbitMQTransportInfrastructure.cs

```diff
@@ -25,13 +25,24 @@ public RabbitMQTransportInfrastructure(SettingsHolder settings, string connectio
             var connectionConfiguration = new ConnectionStringParser(settings).Parse(connectionString);
             connectionFactory = new ConnectionFactory(connectionConfiguration);
 
+            SetUsePublisherConfirmsIfPresent(settings, connectionConfiguration);
+
             CreateTopology();
 
             channelProvider = new ChannelProvider(connectionFactory, routingTopology, connectionConfiguration.UsePublisherConfirms);
 
             RequireOutboxConsent = false;
         }
 
+        static void SetUsePublisherConfirmsIfPresent(SettingsHolder settings, ConnectionConfiguration connectionConfiguration)
+        {
+            if (settings.HasSetting(SettingsKeys.UsePublisherConfirms))
+            {
+                var publisherCofirms = settings.Get<bool>(SettingsKeys.UsePublisherConfirms);
+                connectionConfiguration.UsePublisherConfirms = publisherCofirms;
+            }
```

Comment:
So It might be a good oportunity for another refactoring PR to move default values into SettingsHolder as defaults. But for now I would say that the discussed code make sense. What do you think @adamralph ?